subgraph/queries/harvests.py

```diff
@@ -1,29 +1,31 @@
 from gql import gql
 from rich.console import Console
+from web3 import Web3
 
 from subgraph.subgraph_utils import make_gql_client
 
 console = Console()
 
-harvests_client = make_gql_client("harvests-ethereum")
-
 
 def fetch_tree_distributions(start_timestamp, end_timestamp, chain):
-    tree_client = make_gql_client(f"harvests-{chain}")
+    tree_client = make_gql_client(chain)
     query = gql(
         """
         query tree_distributions(
```

Comment:
![Screenshot 2022-01-05 at 18 54 58](https://user-images.githubusercontent.com/7059697/148257053-3fba8fcb-76b5-45b4-95f2-1e6154adae2d.png)
Syntax checker is complaining about wrong type here. Is it ok ?

Reply:
Maybe it should be `BadgerTreeDistribution_filter` instead of `TokenDistribution_filter` ?

Reply:
Yes that is correct, this was due to using a slightly different subgraph